CONTRIBUTING.md

```diff
@@ -16,5 +16,10 @@ To run the build and tests:
 ./gradlew build
 ```
 
+To publish it:
```

Comment:
I'd suggest keeping this doc internal only (notion).
No one outside of Sentry will be publishing these.